core/store/orm/config_test.go

```diff
@@ -160,75 +158,3 @@ func TestStore_urlParser(t *testing.T) {
 		})
 	}
 }
-
-func TestConfig_NormalizedDatabaseURL(t *testing.T) {
-	t.Parallel()
-
-	tests := []struct {
-		name, uri, expect string
-	}{
-		{"default", "", "/root/db.sqlite3"},
-		{"root", "/root/db.sqlite3", "/root/db.sqlite3"},
-		{"windows root", `C:\root\db.sqlite3`, `C:\root\db.sqlite3`},
-		{"garbage", "89324*$*#@(=", "89324*$*#@(="},
-		{"relative path", "store/db/here", "store/db/here"},
-		{"file uri", "file://host/path", "file://host/path"},
-		{"postgres uri", "postgres://bob:secret@1.2.3.4:5432/mydb?sslmode=verify-full", "postgres://bob:secret@1.2.3.4:5432/mydb?sslmode=verify-full"},
```

Comment:
Should we keep this test around to test the two styles of Postgres URL?

Reply:
Looks like I removed this one by accident, good catch

Reply:
Ah! I remember now. We don't need to care about this function, because it passed through postgres URLs unchanged anyway.